### PR TITLE
Charly/query freshness

### PR DESCRIPTION
**Pull request by CharlyF**

### CODE REVIEW

File: query_freshness.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+initialize(**options)
+now = int(time.time())
+
+query = 'system.cpu.system{*}' # Modify the metric you want to track the freshness of
+
+metric = api.Metric.query(start=now - 3600, end=now, query=query) # how far back in time you want to go (the default is one hour)
+
+nb_points = len(metric['series'][0]['pointlist']) - 1
+last_submission = int(metric['series'][0]['pointlist'][nb_points][0]) / 1000
```
Comment (stephenlechner) [inline query_freshness.py]: Good stuff! Could this be simplified maybe like this?:

```
int(metric['series'][0]['pointlist'][-1][0]) / 1000
```

a la [this stackoverflow](http://stackoverflow.com/a/930398/5540166)

**Comment by stephenlechner** [comment]
that's my only suggestion, and it really doesn't matter--would only cut things down a little bit. 

looks 👍

**Comment by CharlyF** [comment]
@ckelner makes sense. I initially built it alongside with this KB
https://help.datadoghq.com/hc/en-us/articles/115001360786-How-to-track-the-freshness-of-a-metric-
Worth duplicating the content in there?

**Comment by ckelner** [comment]
Hey @CharlyF maybe just a reference to the KB would be good enough. People may discover this code without having come from the KB so giving them a frame of reference would be good IMO.

**Comment by ckelner** [comment]
I did this in https://github.com/DataDog/Miscellany/pull/38

**Pull request merged**